projects/ngrx.io/content/guide/store/action-groups.md

```diff
@@ -0,0 +1,61 @@
+# Action Groups
+
+Action groups is a feature that allows you to group actions together. While writing actions, the actions in most of the cases looks like below.
+
+```typescript
+import { createAction, props } from '@ngrx/store';
+
+export const opened = createAction('[Products Page] Opened');
+
+// defining an action with payload using the `props` function
+export const paginationChanged = createAction(
+  '[Products Page] Pagination Changed',
+  props<{ page: number; offset: number }>()
+);
+
+// defining an action with payload using the props factory
+export const queryChanged = createAction(
+  '[Product Page] Query Changed',
+  (query: Error) => ({ query })
+);
+```
+
+With the help of this new `createActionGroup` api, we can avoid the above code and write the following code.
+
+```typescript
+
+import { createActionGroup } from '@ngrx/store';
+
+export const productPageActions = createActionGroup({
+  source: '[Product Page]',
+  events: {
+    'Pagination Changed': props<{ page: number; offset: number }>(),
+    'Query Changed': (query: Error) => ({ query }),
+  }
+});
+
+```
```

Comment:
Square brackets are not needed for the action group source. I also suggest adding the missing `Opened` action and comments about events:

```suggestion
<code-example header="products-page.actions.ts">
import { createActionGroup, emptyProps, props } from '@ngrx/store';

export const ProductsPageActions = createActionGroup({
  source: 'Products Page',
  events: {
    // defining an event without payload using the `emptyProps` function
    'Opened': emptyProps(),
    
    // defining an event with payload using the `props` function
    'Pagination Changed': props&lt;{ page: number; offset: number }&gt;(),
    
    // defining an event with payload using the props factory
    'Query Changed': (query: string) => ({ query }),
  }
});
</code-example>
```

Reply:
Done